shared/services/beacon/client/std-http-client.go

```diff
@@ -313,6 +313,13 @@ func (c *StandardHttpClient) GetValidatorStatuses(pubkeys []types.ValidatorPubke
 
 // Get whether validators have sync duties to perform at given epoch
 func (c *StandardHttpClient) GetValidatorSyncDuties(indices []string, epoch uint64) (map[string]bool, error) {
+	// Map to store results
+	validatorMap := make(map[string]bool)
+
+	// Return if there are not validators to check
+	if len(indices) == 0 {
+		return validatorMap, nil
```

Comment:
eh, better practice to return `nil, nil` when there's no error but no data, and make sure upstream callers to the function are nil-safe.